sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/DigitalTwinModelData.java

```diff
@@ -0,0 +1,199 @@
+package com.azure.digitaltwins.core.models;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.digitaltwins.core.implementation.models.ModelData;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.time.OffsetDateTime;
+import java.util.Map;
+
+/**
+ * The DigitalTwinModelData representing the model and its corresponding metadata.
+ */
+@Fluent
+public class DigitalTwinModelData {
+
+    private final ObjectMapper mapper;
+
+    /**
+     * Creates a new instance of DigitalTwinModelData.
+     * @param modelData The model data.
+     */
+    public DigitalTwinModelData(ModelData modelData) {
+        mapper = new ObjectMapper();
+
+        this.setId(modelData.getId())
+            .setUploadTime(modelData.getUploadTime())
+            .setDisplayName(modelData.getDisplayName())
+            .setDescription(modelData.getDescription())
+            .setDecommissioned(modelData.isDecommissioned())
+            .setModel(mapper.convertValue(modelData.getModel(), String.class));
```

Comment:
this gives us correctly formatted json string, correct?

Reply:
no, I had to use writeValueAsString to get that out of the object.